### PR TITLE
[DOCFIX] Fix S3 URI for bootstrap script

### PR DESCRIPTION
**Pull request by ns1123**

### CODE REVIEW

**Comment by ZacBlanco** [comment]
Flaky tests

```

**Comment by ns1123** [comment]
Jenkins, test this please

**Comment by ns1123** [comment]
@ZacBlanco can you help to cherry-pick this to branch-2.0?

File: docs/en/compute/AWS-EMR.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ $ aws emr create-default-roles
 ```
 2. The Alluxio bootstrap script is hosted in a publicly readable
 [S3 bucket](https://alluxio-public.s3.amazonaws.com/2.0.1/emr/alluxio-emr.sh).
-This bucket can also be accessed using it's S3 URI: `s3://alluxio-public/emr/alluxio-emr.sh`
+This bucket can also be accessed using it's S3 URI: `s3://alluxio-public/emr/2.0.1/alluxio-emr.sh`
```
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]: Can you double check the path? It looks like it's different from the http URL

Comment (ns1123) [reply docs/en/compute/AWS-EMR.md]: Good eye. Fixed now. Quick sweep shows that was the only one out of order.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Pull request merged**